en/power_module/cuav_can_pmu.md

```diff
@@ -0,0 +1,79 @@
+# CAUV CAN PMU
```

Comment:
@cuhome What flight controllers do you recommend this for/know it will work with? i.e. I am sure it is designed for the CUAV v5+? Have you tested it on others?

Does it come with needed cables to connect to particular flight controllers? If so, which?

Reply:
The pm connector is JST GH.125, which has different lines depending on the flight control (pixhack v3, pixhawk, v5 series flight control); it is not designed for v5+, it can be used on all pixhawk series flight control (Some pixhawks need to modify the wire, such as pixhawk2.1 and pixhawk4). But it is included in the v5+ kit.